src/components/TypeComponents/SimpleSearch/Form/filters/SimpleSearchDateFilterField.js

```diff
@@ -29,7 +29,10 @@ const SimpleSearchDateFilterField = ({
   const { values } = useFormState();
   const { change } = useForm();
 
-  const isSetOrUnset = get(values, `${name}.comparator`) === 'isNull' || get(values, `${name}.comparator`) === 'isNotNull';
+  const noValueNeeded = get(values, `${name}.comparator`) === 'isNull' 
```

Comment:
This is good, would you mind indenting these lines though please to make sure it's clear they're a single condition? :)

Reply:
ok

Reply:
Just nitpicking here. Can we use optional chaining syntax here instead of lodash get?
```suggestion
  const noValueNeeded = values?.[name?.comparator] === 'isNull' 
```